.travis.yml

```diff
@@ -22,6 +22,8 @@ before_install:
     # packages for tests
     - sudo apt-get install --yes -qq parted lsscsi ksh attr acl nfs-kernel-server fio
     - sudo apt-get install --yes -qq libgtest-dev cmake
+    # packages for debugging
+    - sudo apt-get install gdb
```

Comment:
nice, I assume that this is to help us debug problems using gdb on travis directly, correct?

Reply:
right @jkryl .. after this, we need to get a script that loads and prints stacktrace of a coredump